components/new-contribution-flow/ContributorCardWithTier.js

```diff
@@ -83,7 +92,10 @@ const ContributorCardWithTier = ({ collective, contribution, ...props }) => {
             <FormattedMessage
               id="NewContributionFlow.CollectiveAndTier"
               defaultMessage="{collective} - {tier}"
-              values={{ collective: collective.name, tier: contribution.exampleTier }}
+              values={{
+                collective: collective.name,
+                tier: contribution.tier?.name || intl.formatMessage(messages.contributor),
```

Comment:
Just FYI, using a `FormattedMessage` instead of `intl.formatMessage` would have been fine here (and preferred, according to https://github.com/opencollective/opencollective/blob/f9e85996c713672ff95656af0d499c90c0916462/rfcs/010-react-intl.md#L11)

Reply:
A formatted message within a formatted message? 🤯